Gemfile.lock

```diff
@@ -182,7 +177,7 @@ GEM
       railties (>= 3.0.0)
     faker (1.7.3)
       i18n (~> 0.5)
-    ffi (1.11.1)
+    ffi (1.11.3)
```

Comment:
### [ffi](https://github.com/ffi/ffi/wiki)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.11.1 → 1.11.3

[_[change-log](https://github.com/ffi/ffi/blob/master/CHANGELOG.md), [source-code](https://github.com/ffi/ffi/)_]





<details>
<summary>Commits</summary>

A change of **25** commits. See the full changes on [the compare page](https://github.com/ffi/ffi/compare/1.11.1...1.11.3).

These are the first 10 commits:
- (1830511) [Add a notice about yanked ffi-1.11.0](https://github.com/ffi/ffi/commit/1830511ae2678c8d0296a4efe3d204d324962810)
- (aa9df19) [Update ffi.gemspec with correct homepage URL](https://github.com/ffi/ffi/commit/aa9df19f9d6e94f1ea84677bb0c52ffa6a0b88f6)
- (6e4aa91) [Merge pull request #703 from jcn/patch-1](https://github.com/ffi/ffi/commit/6e4aa914d706359d6eb521a11ad5eef828ee56b4)
- (4a902b9) [Fix BigDecimal.new](https://github.com/ffi/ffi/commit/4a902b95e825068c68fde6208df6215ebc48c4d1)
- (d9e07ce) [Merge pull request #711 from wonda-tea-coffee/fix-bigdecimal…](https://github.com/ffi/ffi/commit/d9e07ce7085ca12d0b54d78f15c05f2e64630247)
- (10c8096) [Sort all types.conf files, so that changes are more visible](https://github.com/ffi/ffi/commit/10c8096e81bdfb1d0efbc229563ddb17806ba229)
- (abab76f) [Update platform x86_64-linux on Ubuntu-18.04](https://github.com/ffi/ffi/commit/abab76f7e216bd6211cc74305addf52552fa7280)
- (368d38b) [Describe "types_conf" so that it's shown in "rake -T"](https://github.com/ffi/ffi/commit/368d38b00b925860bfb8df418e20602494efa325)
- (753d522) [Add ruby-2.6 to Appveyor CI](https://github.com/ffi/ffi/commit/753d522ed67fc57f927ae825ae7b9cb8004b5856)
- (8bcf60b) [Fix compiler warnings raised by ruby-2.7](https://github.com/ffi/ffi/commit/8bcf60b622d753f80fd67c7cc9a089c8dd323251)


</details>